src/features/steps/develop/OpenVsCode.tsx

```diff
@@ -62,8 +83,24 @@ export default () => {
                 <Next
                     label="Open VS Code with extension"
                     onClick={() => {
+                        console.log(
```

Comment:
<!-- Polish comments are like a suggestion, where there is nothing necessarily wrong with the relevant content, there's just some ways to immediately improve the quality.  -->
**polish:** ‏Can instead use the logger from shared here now.

Reply:
That log was just a development residue. Thanks for noticing, I remove it.